app/src/main/AndroidManifest.xml

```diff
@@ -22,16 +23,24 @@
 		android:icon="@drawable/ic_launcher"
 		android:label="@string/app_name"
 		android:supportsRtl="true"
+		android:usesCleartextTraffic="true"
```

Comment:
The app doesn't use networking (right now) and if it did it probably wouldn't allow any HTTP traffic, so this option is not needed.